app/views/tv/index.scala

```diff
@@ -44,7 +45,8 @@ object index {
           url = s"$netBaseUrl${routes.Tv.onChannel(channel.key)}"
         )
         .some,
-      robots = true
+      robots = true,
+      playing = true
```

Comment:
`playing = true` causes the zen toggle to appear in the dasher and the top left.

The only other thing this affects is that clicking `Preferences` in the dasher while in lichessTV will open the settings page in a new tab (normally settings page opens in the same tab unless you're in a game). 

I think this change actually makes sense if you're watching lichessTV but lmk if not and I can work around this.